src/nplinker/metabolomics/gnps/gnps_downloader.py

```diff
@@ -53,7 +58,7 @@ def gnps_format(self) -> GNPSFormat:
         """
         return self._gnps_format
 
-    def download(self) -> Self:
+    def download(self) -> "Self":
```

Comment:
Why are the double quotation marks needed?

Reply:
Without the double quote, there is always a warning on `Self` in vscode, it looks vscode cannot recognise it.